server/models/address.ts

```diff
@@ -125,9 +136,7 @@ export default (
       }
     },
     scopes: {
-      withPrivateData: {
-        attributes: {}
-      }
+      withPrivateData: {}
```

Comment:
Basically certain fields are marked private, so when we export the database (with `yarn dump-db`) we don't also export all of the emails, login tokens, etc, because that's a security risk. However sometimes we need to access private fields in our routes, so the withPrivateData scope needs to somehow include all that private data. Maybe sequelize 5 or 6 changed how this is handled. We just need to confirm that queries using withPrivateData still work as expected.

Reply:
oh ok so we need to add `exclude` with the list of field we don't want